### PR TITLE
chore: fix `<Slider />` description info and slider handle overlap issue

### LINKED ISSUE #43731: The description overlaps the handle in the Slider.
### Reproduction link

[https://stackblitz.com/edit/nextjs-vgdl43?file=pages%2Findex.js](https://stackblitz.com/edit/nextjs-vgdl43?file=pages%2Findex.js)

### Steps to reproduce

```tsx
<ConfigProvider
      theme={{
        components: {
          Slider: {
            railSize: 11,
            dotSize: 13,
            controlSize: 20,
            handleSize: 20,
            handleSizeHover: 16
          },
        },
      }}
    >
      <Slider marks={marks} />
    </ConfigProvider>
```

### What is expected?

1. The description should not overlap with the handle in the Slider.
2. Need a property that would add padding between the description and the handle.

### What is actually happening?

The description overlaps the handle in the Slider.

https://github.com/ant-design/ant-design/assets/803674/70ecac1a-6330-41f8-8ab0-377f44dec4af

I tried different property configurations and even added explicit margins to the marks, but it didn't solve the problem.

| Environment | Info |
| --- | --- |
| antd | 5.7.2 |
| React | 18 |
| System | macOS |
| Browser | All |

### PR DESCRIPTION
**Pull request by Wxh16144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

fix #43731

#### Befter

https://github.com/ant-design/ant-design/assets/32004925/ab4989d2-37d8-414f-95d8-fd38ea9bce10

#### After

https://github.com/ant-design/ant-design/assets/32004925/df3e8fd6-6600-4576-83c9-36e22c702f79

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      fix `<Slider />` description info and slider handle overlap issue    |
| 🇨🇳 Chinese |     修复 `<Slider />` 描述信息和滑块手柄重叠问题      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at e259ea4</samp>

Fixed a slider bug where marks overlapped with handle. Adjusted `markInset` in `components/slider/style/index.tsx` to use `railSize` and `token.marginFull`.

### <samp>🤖 Generated by Copilot at e259ea4</samp>

* Fix mark position on slider rail to avoid overlap with handle ([link](https://github.com/ant-design/ant-design/pull/43780/files?diff=unified&w=0#diff-9b9560a611e7ed0e6ef24ca9f1faff1e8c816d3f35ed6a1f73c36d2b42790abaL314-R315))

### CODE REVIEW

**Comment by Wxh16144** [comment]
image

image

**Review by zombieJ — APPROVED** [review]

**Comment by zombieJ** [comment]
changelog 改一下，这个不够语义化哈~

**Comment by zombieJ** [comment]
另外 Argos 变了，需要确认一下是不是预期的。理论上尺寸不应该变才对。

**Comment by Wxh16144** [comment]
> 另外 Argos 变了，需要确认一下是不是预期的。理论上尺寸不应该变才对。

这个需要 reviewers 帮忙看一下,  我感觉 垂直状态下原本就是重叠了. 水平模式下,之前看起来没重叠.但是计算上, top 少了 2px

File: components/slider/style/index.tsx [hunk 1]
```diff
@@ -311,7 +311,8 @@ const genDirectionStyle = (token: SliderToken, horizontal: boolean): CSSObject =
       // Reset all
       insetInlineStart: 0,
       top: 0,
-      [markInset]: handleSize,
+      // https://github.com/ant-design/ant-design/issues/43731
+      [markInset]: railSize * 3 + (horizontal ? 0 : token.marginFull),
```
Comment (Wxh16144) [inline components/slider/style/index.tsx:315]: ## 1. 为什么是 railSize * 3 ?

padding-block  + height =  railSize * 2 + railSize = railSize * 3

image

## 2. 为什么垂直情况下还需要加 marginFull ?

image

**Pull request merged**